clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -358,6 +452,15 @@ public int sessionId() {
                 .setResponses(topicResponseList);
     }
 
+    private Boolean supportsTopicIds() {
```

Comment:
Shouldn't we be using versions and expect non-zero ids in new versions?

Reply:
I had trouble getting the version into the response. The constructor is used in some places where we don't have access to the version.

Reply:
AbstractResponse does not maintain version like AbstractRequest. So I'm not sure the best way to proceed with this.